### ISSUE TITLE
http://localhost:3009/companies  return NotImplementedError

### ISSUE THREAD

**Issue by tablecell** [restocat/restocat-example#2]
restocat-example\collections\index\collections.json
>   "logic": "index.js",

index.js
```A
  list() {
    return {
      '/': 'index',
      '/companies': {
        get: 'List of companies',
        post: 'Create new company'
      },

```

**Comment by chetverikov** [comment]
Could you write a request which returns "NotImplementedError"? What method did you use POST or GET?

**Comment by tablecell** [comment]
1. git clone https://github.com/restocat/restocat-example
2. cd restcoat-example  
3. npm install 
4. node server.js 
5.  client get request
> curl http://localhost:3009/companies
{"name":"NotImplementedError","status":501,"message":"Resource or collection '/companies' not implemented in API","code":"NotImplemented.
6. server console 
> [2019-12-01 17:46:21.321] [ERROR] SYSTEM   - { NotImplementedError: Resource or collection '/companies' not implemented in API
    at RequestRouter.notImplementedHandle ( restocat-example\node_modules\restocat\lib\router\RequestRouter.js:221:11)
    at RequestRouter.requestListener ( restocat-example\node_modules\restocat\lib\router\RequestRouter.js:75:27)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:118:7)
  name: 'NotImplementedError',
  status: 501,
  message: 'Resource or collection \'/companies\' not implemented in API',
  code: 'NotImplemented' }
[2019-12-01 17:46:21.384] [INFO] RESPONSE - [302e6214-298b-835d-88f2-930247be1c9e] ::ffff:127.0.0.1 - - (82ms) GET http://localhost:3009/companies HTTP/1.1 501 Not Implemented

**Comment by chetverikov** [comment]
It seems you changed some file in an example project because I reproduced your flow and I didn't get a response like yours. Could you show me your ./routes.js and ./collections/companies/collection.json

**Comment by tablecell** [comment]
> 
> 
> It seems you changed some file in an example project because I reproduced your flow and I didn't get a response like yours. Could you show me your ./routes.js and ./collections/companies/collection.json

routes.js  is same as https://github.com/restocat/restocat-example/blob/master/routes.js
```JS
module.exports = {
index: '/',
companies: '/companies',
reviews: '/reviews'
};
```
 ./collections/companies/collection.json is  same as https://github.com/restocat/restocat-example/blob/master/collections/companies/collection.json
```JS
{
"name": "companies",
"logic": "index.js",
"serviceName": "companiesService",
"endpoints": {
"get /:id/reviews": "reviews"
}
}
```
after clone code from github I didn't change any code    
my environment is win7 64bit  node version is v9.8.0   npm version is 5.6

**Comment by chetverikov** [comment]
The bug has been fixed in 4.0.1. Please pull new changes for restocat-example and update node modules
